.github/workflows/ci.yml

```diff
@@ -17,8 +17,11 @@ jobs:
         with:
           node-version: "16.x"
           cache: "npm"
-      - name: Install dependencies
+      - name: Install website dependencies
         run: npm ci
+      - name: Install playground dependencies
+        working-directory: src/playground
+        run: npm install
```

Comment:
```suggestion
        run: npm ci
```

Reply:
`npm ci` works only if we have `package-lock.json`.

Reply:
We already have an `install:playground` script which we can use.

https://github.com/eslint/eslint.org/blob/38d7f6cdd4651ae0077f6611b06ab2d3d8cf04bf/package.json#L18

Reply:
sure